lib/compiler/test/beam_doc_SUITE_data/converted_metadata.erl

```diff
@@ -0,0 +1,8 @@
+-module(converted_metadata).
+
+-moduledoc(#{ authors => ["me","myself","I"], format => "custom", since => "1.0", deprecated => "yes" }).
+
+-export([test/0]).
+
+-doc #{ since => "1.0", deprecated => "yes", equiv => "other", group => "collection" }.
+test() -> ok.
```

Comment:
```suggestion
# SPDX-License-Identifier: Apache-2.0
# SPDX-FileCopyrightText: 2024 Erlang/OTP and contributors
-module(converted_metadata).

-moduledoc(#{ authors => ["me","myself","I"], format => "custom", since => "1.0", deprecated => "yes" }).

-export([test/0]).

-doc #{ since => "1.0", deprecated => "yes", equiv => "other", group => "collection" }.
test() -> ok.
```